client/app/app.vue

```diff
@@ -26,7 +26,9 @@
     @import '../node_modules/normalize.css/normalize.css'
     @import './styles/base/_normalize'
     @import './styles/base/_colors'
+
     @import 'cozy-ui'
+    @import './styles/base/_dialog'
```

Comment:
As long as the `_dialog.styl` stylesheet contains rulesets that apply to the dialog component, they should be scoped in the `dialog.vue` file.